content/en/news/releases/1.11.x/announcing-1.11.2/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Announcing Istio 1.11.2
+linktitle: 1.11.2
+subtitle: Patch Release
+description: Istio 1.11.2 patch release.
+publishdate: 2021-09-02
+release: 1.11.2
+aliases:
+    - /news/announcing-1.11.2
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.11.1 and Istio 1.11.2.
+
+{{< relnote >}}
+
+## Changes
+
+- **Improved** `istioctl install` to give more details during installation failures.
+
+- **Added** two mutually-exclusive flags to `istioctl x workload entry configure`
+    - **`--internal-ip`** configures the VM workload with a private IP address used for workload auto registration and health probes.
+    - **`--external-ip`** configures the VM workload with a public IP address used for workload auto registration. Meanwhile, it configures health probes to be performed through localhost. By setting the environment variable `REWRITE_PROBE_LEGACY_LOCALHOST_DESTINATION` to true.
```

Comment:
What does "By setting the environment variable `REWRITE_PROBE_LEGACY_LOCALHOST_DESTINATION` to true." mean? Does the earlier configure health problem only happen if the env variable is true?